packages/babel-plugin-replace-styles/src/index.ts

```diff
@@ -26,26 +26,45 @@ export default function replaceStyles(
   { types }: Babel,
   { scope, use, replace = 'main' }: ReplaceStylesOptions
 ): babel.PluginObj {
-
   /**  Replace an import inside a component */
   function replaceCssImports(
     path: NodePath<BabelTypes.ImportDeclaration>,
     state: PluginPass,
     importName: string
   ) {
-    const isScope = state?.file?.opts?.filename && state.file.opts.filename.includes(`@${scope}/`);
+    const isScope =
+      state?.file?.opts?.filename &&
+      state.file.opts.filename.includes(`@${scope}/`);
 
     if (isScope && importName.includes(`${replace}.css`)) {
       // We found a candidate for replacement
       const dirname = nodePath.dirname(state.file.opts.filename as string);
-      const next = nodePath.join(dirname,importName.replace(replace, use));
-      
+      const next = nodePath.join(dirname, importName.replace(replace, use));
+
       if (exists(next)) {
         // Replacement exists
-        const importDeclaration = types.importDeclaration([], types.stringLiteral(`../${use}.css`));
+        const importDeclaration = types.importDeclaration(
+          [],
+          types.stringLiteral(`../${use}.css`)
+        );
         path.replaceInline(importDeclaration);
       }
-      
+    }
+  }
+
+  /**  Replace an css.js import inside a component */
+  function replaceCssJsImports(
+    path: NodePath<BabelTypes.ImportDeclaration>,
+    state: PluginPass,
+    importName: string
+  ) {
+    const isScope =
+      state?.file?.opts?.filename &&
+      state.file.opts.filename.includes(`@${scope}/`);
+
+    if (isScope && path.node.specifiers.length) {
+      // eslint-disable-next-line no-param-reassign
+      path.node.source.value = importName.replace('.css', `-${use}.css`)
```

Comment:
Should this also check if the file exists before swapping, like the CSS one?

Reply:
good idea